src/front/DataInspector.js

```diff
@@ -2,7 +2,18 @@ L.TileLayer.XRay = L.TileLayer.extend({
 
     getTileUrl: function (tilePoint) {
         this.options.version = Date.now();
-        this.options.showLayer = L.K.Config.dataInspectorLayer || '';
+        var keys = Object.keys(L.K.Config);
+        if (L.K.Config['dataInspectorLayer___all__'] === true) {
```

Comment:
@yohanboniface I am trying to do this in lines 70-72 here: https://github.com/kosmtik/kosmtik/pull/35/files#diff-f298892263f8b31e1b116a56600a5644R70 but while it is setting up the UI, I get a JS error:

```Uncaught TypeError: Cannot read property '__all__' of undefined - Leaflet.FormBuilder Line 81```